com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoop.cs

```diff
@@ -121,18 +401,29 @@ class LightDefinitions
         public static uint s_ScreenSpaceColorShadowFlag = 0x100;
         public static uint s_InvalidScreenSpaceShadow = 0xff;
         public static uint s_ScreenSpaceShadowIndexMask = 0xff;
+
+        // Binned lighting
+        // For performance reasons, keep all sizes in powers of 2.
+        public static int s_CoarseTileEntryLimit = 64; // Per category; before pruning, so the number is lower in practice; the number could be made (almost always) exact at the cost of making the 'FillCoarseTiles' pass more complicated
```

Comment:
From the comment I do'nt understand what the 64 and 16 imply here, what is the limit?

Reply:
Coarse tiles are 64x64 pixels. Fine tiles are 8x8 pixels.
Each coarse tiles can contain at most 64 indices per category (64 punctual & 64 area & ..., etc). Fine tiles can contain 16 indices.
Since the coarse tiles are computed in two steps (fill and prune), it may not be possible to have 64 entries there (some will be pruned). We can use extra storage for indices before the pruning but this hasn't been implemented yet.